packages/eslint-plugin-react-hooks/src/ReactiveDeps.js

```diff
@@ -376,7 +378,11 @@ export default {
           ' ' +
           (set.size > 1 ? 'dependencies' : 'dependency') +
           ': ' +
-          join(Array.from(set).map(quote)) +
+          join(
+            Array.from(set)
+              .sort()
+              .map(quote),
+          ) +
```

Comment:
Maybe this could also be space for a future "sorted-dependencies" rule? 🤔

Reply:
One simple compromise would be to only add at the end by default, but alphabetize if the current array is empty.

Reply:
Actually we could alphabetize if existing list is alphabetized. 😏